### ISSUE TITLE
Avez-vous besoin de restreindre le géocodage à un territoire ?

### ISSUE THREAD

**Issue by jfgigand** [jfgigand/geocodage#3]
Dans votre contexte d'utilisation, avec-vous fréquemment besoin de géocoder une adresse en la restreignant à un territoire administratif (commune, département...) ?

Ce filtrage doit-il être strict ou n'est-il qu'une préférence de recherche (pour des adresses sans commune, par exemple).

Ce besoin serait-il plus valable pour le géocodage de masse (batch geocoding) ?

**Comment by ebuard** [comment]
oui, géocoage strict. Par exemple, france entière, il existe plusieurs communes de Bordeaux, alors qu'on pourrait déjà limiter à un département.

**Comment by cquest** [comment]
En batch, le géocodeur BAN donne les meilleurs résultats lorsqu'on utilise un filtre sur le code INSEE de la commune.
C'est une information qu'on a souvent dans les bases de données et qui permet d'éviter de chercher une adresse ailleurs que dans la commune déjà identifiée.
On a aussi la possibilité de filtrer sur le code postal, mais celui-ci peut poser problème sur les CEDEX/BP/CS avec des codes postaux spécifiques absents de la BAN.
